zerver/lib/actions.py

```diff
@@ -5410,20 +5410,26 @@ def send_change_stream_message_retention_days_notification(
     with override_language(stream.realm.default_language):
         if old_value == Stream.MESSAGE_RETENTION_SPECIAL_VALUES_MAP["unlimited"]:
             notification_string = _(
-                "{user} has changed the [message retention period](/help/message-retention-policy) for this stream from "
-                "**Forever** to **{new_value} days**. Messages will be automatically deleted after {new_value} days."
+                "{user} has changed the [message retention period](/help/message-retention-policy) for this stream: \n"
```

Comment:
I think it may be worth splitting out the summary line and the message before that, to avoid repeating very similar strings 3 times?

Reply:
Thank you @punchagan. I have added a commit to refactor the code and I think using your suggested approach is helpful :pray: